test/drop_test/ffi.py

```diff
@@ -125,7 +125,10 @@ async def wait_for_any_event(self, duration: int, ignore_progress: bool = False)
         for _ in range(0, duration):
             with self._lock:
                 self._events = [
-                    ev for ev in self._events if not isinstance(ev, event.Throttled)
+                    ev
+                    for ev in self._events
+                    if not isinstance(ev, event.Throttled)
+                    and not isinstance(ev, event.TransferDeferred)
```

Comment:
I am marking this place but it refers to other places ignoring Deferred event as well - `TransferDeferred` looks pretty deterministic so we could actually not ignore it but instead alter the testcases to look for it. it should be a pretty easy change as well. What is your take on this? :)

Reply:
I can try not to ignore it but I fear it will blow up test cases.  The thing is that it may be emitted an unspecified amount of times before making a connection so it may break waiting for `TransferStart` events

Reply:
You're totally right about this one but I have a feeling it should be fine, it doesn't retry without a delay, very curious to see if it's possible. This would result in nicer testrunner as well, those ignore parameters keep piling up :))

Reply:
Removed it and added a configurable retry count